catalog/designers/nontynet/info.pb

```diff
@@ -0,0 +1,5 @@
+designer: "Nontynet"
```

Comment:
I'd prefer `Nonty` to this, what do you think @mintoming ?

Reply:
Name change pushed below :)